src/main/cli/settings.c

```diff
@@ -375,11 +375,10 @@ static const char * const lookupOverclock[] = {
     };
 #endif
 
-static const char * const lookupTableThrottleLimitType[] = {
+const char * const lookupTableThrottleLimitType[] = {
```

Comment:
```suggestion
static const char * const lookupTableThrottleLimitType[] = {
```

Reply:
Can't make it static, when it extern in H files